compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -1119,8 +1119,8 @@ fn generic_simd_intrinsic<'ll, 'tcx>(
     if name == sym::simd_select_bitmask {
         let (len, _) = require_simd!(arg_tys[1], SimdArgument);
 
-        let expected_int_bits = (len.max(8) - 1).next_power_of_two();
```

Comment:
I don't know why the old code did `-1` here. `next_power_of_two` is already idempotent; when the input is a power of 2 it is returned unchanged.